templates/README.twig

```diff
@@ -59,10 +59,13 @@ brew install {{ brew_package }}
 ### Ubuntu / WinWSL+Ubuntu:
 
 ```
-# Setup repository
-curl -s https://packagecloud.io/install/repositories/{{ debian_apt_repo }}/script.deb.sh | bash
-# Install package
-{{ debian_package_install | raw }}
```

Comment:
I think we should keep the APT instructions in place in this PR, and merge once the package is available in an apt repo, hopefully in early January.